src/components/NavBar.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+
+import {
+  FilmIcon,
+  HomeIcon,
+  QueueListIcon,
+  UserIcon,
+} from "@heroicons/react/24/solid";
+
+export function NavBar() {
+  const [icon, setIconState] = useState("home");
+
+  return (
+    <div className="bg-dark flex pl-16 pr-16 pt-8 pb-8 gap-6 justify-evenly items-center">
+      <HomeIcon
```

Comment:
You could create an array of navItems and map over it instead of repeating the element